src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -10,7 +10,7 @@
 
 from azure.cli.core.commands.parameters import get_enum_type, get_three_state_flag, get_location_type, tags_type
 from azure.cli.core.commands.validators import validate_file_or_dict
-from azure.cli.core.profiles import ResourceType
+# from azure.cli.core.profiles import ResourceType
```

Comment:
We can remove it directly.

Reply:
We need it back after the swagger is fixed.